features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/components/SearchUserBar.kt

```diff
@@ -99,6 +101,11 @@ fun SearchUserBar(
                 )
             }
         },
+        contentSuffix = {
```

Comment:
I prefer using the content suffix here instead of adding a new state as it's not a common pattern in the SearchBar for now.